src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -198,10 +198,13 @@ public Task StartAsync()
                 // Get a connection ID from the server
                 logger.EstablishingConnection(url);
                 using (var request = new HttpRequestMessage(HttpMethod.Options, url))
-                using (var response = await httpClient.SendAsync(request))
                 {
-                    response.EnsureSuccessStatusCode();
-                    return await ParseNegotiateResponse(response, logger);
+                    request.Headers.UserAgent.Add(SendUtils.DefaultUserAgentHeader);
```

Comment:
It would be good to find a better location of `SendUtils.DefaultUserAgentHeader` now that it is used not only for sending messages.

Reply:
Hmm, where do you think this belongs?

Reply:
Maybe just the `Utils` file?

Reply:
Create a new class (e.g. `Constants`) and put it there.